src/commands/publish/mod.rs

```diff
@@ -166,6 +126,21 @@ fn validate_project(project: &Project, release: bool) -> Result<(), failure::Err
         missing_fields.push("name")
     };
 
+    match &project.kv_namespaces {
```

Comment:
the form 

```
match foo {
    Some(bar) => { stuff }
    None => {}
```

can be written more clearly as

```
foo.map(|bar| { stuff })
```

Reply:
this wasn't intended to be part of this pr, it just crept in when the target branch got rebased.